scripts/webpack/frontend.js

```diff
@@ -5,13 +5,11 @@ const glob = promisify(require('glob'));
 const { smart: merge } = require('webpack-merge');
 const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
 const ReportBundleSize = require('./plugins/report-bundle-size');
-const Progress = require('./plugins/progress');
```

Comment:
Can we delete this plugin, and maybe some of its dependencies?

Reply:
Done.